themes/bootstrap3/templates/RecordDriver/SolrOverdrive/result-list.phtml

```diff
@@ -0,0 +1,185 @@
+              <a href="<?=$this->record($this->driver)->getLink('author', $this->highlight($summAuthor, null, true, false))?>"><?=$this->highlight($summAuthor)?></a><?=$i + 1 < $authorCount ? ',' : ''?>
+            <?php endforeach; ?>
+          <?php endif; ?>
+<?php endif; ?>
+
+      </div>
+
+      <?php if(!$this->driver->isCollection()): ?>
+        <?php if ($snippet = $this->driver->getHighlightedSnippet()): ?>
+          <?php if (!empty($snippet['caption'])): ?>
+            <strong><?=$this->transEsc($snippet['caption']) ?>:</strong> ';
+          <?php endif; ?>
+          <?php if (!empty($snippet['snippet'])): ?>
+            <span class="quotestart">&#8220;</span>...<?=$this->highlight($snippet['snippet']) ?>...<span class="quoteend">&#8221;</span><br/>
+          <?php endif; ?>
+        <?php endif; ?>
+      <?php endif; ?>
+
+      <?php
+      /* Display information on duplicate records if available */
+      if ($dedupData = $this->driver->getDedupData()): ?>
+        <div class="dedupInformation">
+        <?php
+          $i = 0;
+          foreach ($dedupData as $source => $current) {
+            if (++$i == 1) {
+              ?><span class="currentSource"><a href="<?=$this->recordLink()->getUrl($this->driver)?>"><?=$this->transEsc("source_$source", [], $source)?></a></span><?php
+            } else {
+              if ($i == 2) {
+                ?> <span class="otherSources">(<?=$this->transEsc('Other Sources')?>: <?php
+              } else {
+                ?>, <?php
+              }
+              ?><a href="<?=$this->recordLink()->getUrl($current['id'])?>"><?=$this->transEsc("source_$source", [], $source)?></a><?php
+            }
+          }
+          if ($i > 1) {
+            ?>)</span><?php
+          }?>
+        </div>
+      <?php endif; ?>
+
+        <div class="callnumAndLocation ajax-availability hidden">
+      <?php if($this->driver->supportsAjaxStatus()):?>
+          <div class="callnum ajax-availability hidden status">
+              <?=$this->transEsc('Loading')?>...<br/>
+          </div>
+
+      <?php elseif($avail = $this->driver->getOverdriveAvailability()): ?>
+            <div class="availability">
+             <div class='copies'><strong><?= $this->transEsc("od_avail_total")?></strong> <?=$avail->copiesOwned ?></div>
+             <div class='avail'><strong><?= $this->transEsc("od_avail_avail")?></strong> <?=$avail->copiesAvailable ?></div>
+             <div class='holds'><strong><?= $this->transEsc("od_avail_holds")?></strong> <?=$avail->numberOfHolds ?></div>
+            </div>
+           
+       <?php endif; ?>
+    </div>
+           
+           <div class="result-formats">
+             <span class="format book"><?=$this->driver->getFormats()[0]?></span>
+            </div>
+          
+
+
+      <?php /* We need to find out if we're supposed to display an OpenURL link ($openUrlActive),
```

Comment:
There's a lot of commented-out code here; is this work in progress or should it be removed?

Reply:
Oh hmm.  I'm wasn't sure what that code does and whether it's applicable to Overdrive Integration or not.  It looks like it's for embedding metadata?  Based on what you know about this, should I try to support this in the record driver?

Reply:
Regarding the Demo connector idea, it does sound like a lot of work.  It might be worth it if is something that could also somehow be used for unit testing.  I don't think I have it in me to do that right now, but if somebody wanted to do it, I could help with providing sample content.  Especially given how little interest there has been.

Reply:
The OpenURL logic displays links to your link resolver to find full-text content related to the record. I'd think in Overdrive, the need for this would be lessened, but it might still be useful to provide an alternate avenue for obtaining content that is currently checked out.

Also, regarding the Demo connector, I definitely understand your reluctance to build it right now; it definitely seems that it would be a lot of work. Such a connector would definitely be helpful for unit testing, and that's another goal that would be great to meet eventually, but I don't expect you to do all of that work at this stage.

Reply:
Uncommenting the block doesn't seem to break anything for me.  I'll leave it like that for now.  I guess I'm not sure if I need to do anything in my Record Driver to support it if someone configured their system for OpenURLs.

Reply:
I think it should be okay; that logic should be inherited from the parent class. It's possible that some tweaking might be necessary to identify the types of materials correctly, but we can cross that bridge when we come to it.